video/vdu_sprites.h

```diff
@@ -158,6 +158,16 @@ void VDUStreamProcessor::vdu_sys_sprites(void) {
 			debug_log("vdu_sys_sprites: sprite %d - bitmap %d added as frame %d\n\r", getCurrentSprite(), bufferId, getSprite()->framesCount-1);
 		}	break;
 
+		case 0x40: {	// Setup mouse cursor from current bitmap
+			auto hotX = readByte_t(); if (hotX == -1) return;
+			auto hotY = readByte_t(); if (hotY == -1) return;
```

Comment:
technically the "hot" position can use a 16-bit position - but that's not practical - so let's not encourage anyone to make such massive mouse cursors :grin: